README.md

```diff
@@ -97,17 +99,17 @@ If the Docker images are not built yet, this script builds them.
 However, it doesn't automatically build a new images even when the source code is modified. 
 You must rebuild images using `distDocker` gradle task.
 
-### Stopping the test cluster
+### Stopping waltz test cluster
 
-    bin/test-cluster.sh stop
+    bin/test-cluster.sh stop <- stop all created clusters
+    bin/test-cluster.sh stop <cluster_name> <- stop cluster with the given cluster name
 
-This stops all three containers. You can resume the cluster using `start` command. All data in zookeeper and storages are preserved.
+This stops waltz containers. You can resume the cluster using `start <cluster_name>` command. All data in zookeeper and storages are preserved.
 
```

Comment:
nit: maybe we can add the descriptions about test-cluster.sh restart in readme, looks it was missed here.